msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -96,17 +96,9 @@ You can enable or disable [group owner consent](/azure/active-directory/manage-a
 
 In addition, you can enable or disable group owner consent using PowerShell, follow the steps outlined in [configure group owner consent using PowerShell](/azure/active-directory/manage-apps/configure-user-consent-groups?tabs=azure-powershell).
 
-### Configure user consent settings for RSC in a chat
+### Configure RSC in a chat
```

Comment:
Configure chat owner consent settings for RSC in a chat (via Microsoft Graph APIs)